react/week4/github-reasecher/src/context/Lounchpad.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import {  SearchContextProvider } from './createContext';
+import Header from './Header'
+import SearchInput from './SearchInput';
+import UserSearchResults from './UserSearchResults';
+
+const Lounchpad = () => {
+
+  const [users, setUsers] = useState([])
+  const [error, setError] = React.useState();
```

Comment:
Very tiny thing, but since you already imported { useState }, you don't need the React. in front.
```suggestion
  const [error, setError] = useState();
```

Reply:
Thank you, Joachim.
I`ll rewritte